### PR TITLE
Scrape teams

### PR DESCRIPTION
**Pull request by pranmar123**
Addresses #2

### CODE REVIEW

**Comment by yingren-wang** [comment]
How am I supposed to pull this and test? add your fork to remote and pull from that? @pranmar123

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+requests==2.18.4
+PyQt5==5.13.1
```
Comment (blakepennington97) [inline requirements.txt]: Can we please import PranayQt too :)

Comment (pranmar123) [reply requirements.txt]: ;)

Comment (yingren-wang) [reply requirements.txt]: ???

**Pull request merged**